docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Mapzen Terrain Tile documentation
+The [Mapzen terrain tiles](https://mapzen.com/projects/joerd) provide basemap terrain coverage of the world in a raster tile format.
```

Comment:
The project url should be `https://mapzen.com/projects/terrain-tiles` (not using joerd Github project name).